### DISCUSSION TITLE
Just another post to put the old feed back

### DISCUSSION THREAD

**Discussion by AlbanCrepel** [community/community#37709] [Other Feature Feedback, Questions, & Ideas]
Yes, there are plenty of posts about it, this is just my little contribution.

Stop proposing stuff I don't care about, alright ? Don't be like other social network platforms just to discover new stuff, the "explore" section is the right way to do it for people who care. At least, keep some decent alternatives to keep the old feed.

I hope that you all, working at github, are actually reading all these feedbacks and see the mess you have done. In code, we like the KISS principle, well... Just keep your user interface simple and stupid. I follow people and repos, and want to see that, not some things I don't care about. Thank you.

With all my love and respect.

**Comment by ossanna16** [comment]
Hi there @AlbanCrepel and welcome to our community! Thank you for the feedback!

**Comment by sathya-nuvai** [comment]
This plan is now tracked as **Issue [#782](https://github.com/Nuvai/NuvaiCode/issues/782)**. 

Closing this discussion in favor of tracking on the issue. @haripriya — please review on the issue thread.
